packages/discovery-react-components/src/utils/document/processDoc.ts

```diff
@@ -186,14 +186,7 @@ function setupBodyParser(parser: SaxParser, doc: ProcessedDoc): void {
   parser.pushState({
     onopentag: (p: Parser, tagName: string, attributes: Attributes): void => {
       if (SECTION_NAMES.includes(tagName)) {
-        setupSectionParser(
-          parser,
-          doc,
-          tagName,
-          attributes,
-          p.startIndex,
-          getChildBeginFromOpenTag(p)
-        );
+        setupSectionParser(parser, doc, tagName, attributes, getChildBeginFromOpenTag(p), p);
```

Comment:
I think we actually want to pass `p.startIndex` instead of `getChildBeginFromOpenTag(p)` -- the latter is handled within `onopentag`.